src/modules/users/users.controller.ts

```diff
@@ -99,6 +102,11 @@ export class UserController {
 
     await this.usersService.updateUser(updateUserDto);
 
+    this.client.emit(
```

Comment:
this call relates to business logic, I guess

Reply:
moved to users.service.ts